packages/bot-web-ui/src/components/dashboard/tutorial-components/tutorial.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { Tabs, Tab, Text, Accordion, Icon } from '@deriv/components';
+import { connect } from 'Stores/connect';
+import RootStore from 'Stores/index';
+import LoadModalStore from 'Stores/load-modal-store';
+import { localize } from '@deriv/translations';
+import { FAQ_QUESTIONS } from './tutorial_faq';
+
+type DashboardProps = {
+    active_tab_tutotials: number;
+    faq_search_value: string;
+    setActiveTabTutorial: (active_tab_tutotials: number) => void;
+    setFAQSearchValue: (setFAQSearchValue: string) => void;
+};
+
+const Tutorial = ({ active_tab_tutotials, setActiveTabTutorial, setFAQSearchValue }: DashboardProps) => {
+    return (
+        <div className='dc-tabs__wrapper'>
+            <input
+                type='text'
+                placeholder='Search'
+                className='dc-tabs__wrapper__search_input'
+                onChange={e => {
+                    setFAQSearchValue(e.target.value);
+                }}
+            />
+            <Tabs active_index={active_tab_tutotials} onTabItemClick={setActiveTabTutorial} top>
+                {/* [Todo] needs to update tabs comIcDashBoardComponentsTabponent children instead of using label property */}
+
+                <Tab label={localize('Guide')} />
+                <Tab label={localize('FAQ')} id='id-bot-builder'>
+                    <div className='dc-tabs__wrapper'>
+                        <Text as='p' size={'s'} line_height='xl' className='dc-tabs__wrapper__faq_header' weight='bold'>
+                            FAQ
+                        </Text>
+                        <Accordion
+                            className='tutorial-faq'
+                            list={FAQ_QUESTIONS.map(({ title, description }) => {
+                                return {
+                                    header: (
+                                        <Text as='p' size={'s'} line_height='xl' className='faq_title' weight='bold'>
```

Comment:
size s is default. we can remove

Reply:
className='faq_title' is not respecting the convention 
it should be something like:
 className='faq-title' or if title is an element inside faq it should be : `faq__title`
 
 please read this documentation:
 https://getbem.com/naming/

Reply:
alright. fixed